src/pages/get-started/develop/vanilla.mdx

```diff
@@ -158,7 +158,7 @@ Carbon Components are built to be included individually and not clobber global s
 
 ### JavaScript
 
-Carbon Component has component JavaScript classes, each of which correspond to a component found in [our components page](../../components/overview). The first steps to work with component JavaScript classes are the following:
+Carbon Component has component JavaScript classes, each of which correspond to a component found in [our components page](../../../components/overview). The first steps to work with component JavaScript classes are the following:
```

Comment:
We actually wanna not use absolute paths for pages, gatsby handles this for us by just using the page url:

```suggestion
Carbon Component has component JavaScript classes, each of which correspond to a component found in [our components page](/components/overview). The first steps to work with component JavaScript classes are the following:
```

Reply:
@jfrantzius checking in to see if you can update the PR based on @vpicone's comment